x/authz/keeper/msg_server.go

```diff
@@ -38,7 +38,7 @@ func (k Keeper) Grant(goCtx context.Context, msg *authz.MsgGrant) (*authz.MsgGra
 	ctx := sdk.UnwrapSDKContext(goCtx)
 	granteeAcc := k.authKeeper.GetAccount(ctx, grantee)
 	if granteeAcc == nil {
-		if k.bankKeeper.BlockedAddr(grantee) {
+		if k.bankKeeper != nil && k.bankKeeper.BlockedAddr(grantee) {
```

Comment:
i'm not sure what this check guards us against but maybe we'd rather panic if authz doesn't get constructed with a bank keeper? @aljo242

Reply:
Not sure I follow. Why would panicking in an API handler ever be the correct behavior?

A nil bank keeper is the default state on initialization. You can only set it using a follow up method call.

Reply:
We need this check in the keeper and cannot skip it whatsoever.  We should find some way to ensure that you cannot start you app without having the bankkeeper properly set on the authzkeeper

Reply:
My take is that we should just bite the bullet and require the `bankkeeper` to be set when constructing the AuthzKeeper.  The intended behavior now is that you _must_ have this field set on your AuthzKeeper but we are not enforcing that in code

Reply:
You prefer a breaking change to the Keeper constructor then? I can change the PR to that's the ideal solution.

The alternative in my opinion is to just return an error on uninitialized accounts when the bank keeper is not set here. 

Since the app module constructor automatically sets the bank keeper in initialization, I think this failure scenario only really matters when passing the keeper as a dependency to other modules internal to the chain--not for RPC handlers.